### PR TITLE
updating readme, closes #553

### LINKED ISSUE #553: Upgrade dev environment to Matlab R2018b or more recent.
All code should be ported to/supported by a version of Matlab R2018b or more recent.

### PR DESCRIPTION
**Pull request by ocehugo**

### CODE REVIEW

**Comment by ocehugo** [comment]
@ggalibert , please review and merge.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
 
-Along with its source code, binaries are provided so that **it can also be used without Matlab**.
+The IMOS Toolbox aims at **converting oceanographic instrument files into quality-controlled IMOS compliant NetCDF files**. 
 
-It can connect to a deployment database in order to collect the relevant metadata attached to each data set. An MS-Access file is provided as a default deployment database but any type of database can be used. The Java code can use any JDBC API to query the deployment database. By default, [UCanAccess](http://ucanaccess.sourceforge.net/site.html) is used to query the MS-Access file.
+The toolbox process instruments deployed on moorings (**time series**) or during casts (**profiles**). The processing, including quality control (**QC**) of several oceanographic variables, can be done in **batch mode** or **interactive**.
 
-The toolbox can currently process instruments deployed on moorings (**time series**) or during casts (**profiles**).
+Finally, the package allows deployment metadata to be ingested into the files from MS-access databases, CSV files, or added manually through a graphical user interface (GUI). Manual **QC** is also possible.
```
Comment (ggalibert) [inline README.md]: any database can be supported actually

File: README.md [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
 
-The processing of these files can be performed **interactively** with a graphical interface or in **batch mode**.
+See our [wiki page](https://github.com/aodn/imos-toolbox/wiki) page for more details.
 
-Please read the [wiki](https://github.com/aodn/imos-toolbox/wiki) for more information on how to **install** and **use** the toolbox. You can also read the [licence](https://raw.githubusercontent.com/aodn/imos-toolbox/master/license.txt) for copyright and licensing information.
+# Distribution/Requirements
 
-This project is designed and maintained by [ANMN](http://imos.org.au/facilities/nationalmooringnetwork/) and [AODN](http://imos.org.au/facilities/aodn/). It is licensed under the terms of the GNU GPLv3.
+The **stable** releases may be obtained [here](https://github.com/aodn/imos-toolbox/releases). The releases contain both the source code and binary applications (executables).
```
Comment (ggalibert) [inline README.md]: Add information about installation with ongoing support and point to the install page of the wiki, you've removed this link and is important.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
+
+We support Windows and Linux. The toolbox may be used as a **Matlab stand-alone library** or **stand-alone application** (**No Matlab is required**).
+
+For a **stand-alone library** usage, **Matlab R2018b+** is required ( since version 2.6.1).
```
Comment (ggalibert) [inline README.md]: I don't think the correct name of the Matlab version includes a "+"

File: README.md (hunk 1 above)
Comment (ocehugo) [reply README.md]: oops, removed by accident. done

File: README.md (hunk 2 above)
Comment (ocehugo) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (ocehugo) [reply README.md]: done + re-organized.

**Comment by ocehugo** [comment]
@ggalibert , happy with the changes?

I want to merge this before tagging the pre-release as "release".

File: README.md [hunk 4]
```diff
@@ -8,20 +8,22 @@ Finally, the package allows deployment metadata to be ingested into the files fr
...
 
-We support Windows and Linux. The toolbox may be used as a **Matlab stand-alone library** or **stand-alone application** (**No Matlab is required**).
 
-For a **stand-alone library** usage, **Matlab R2018b+** is required ( since version 2.6.1).
+# Requirements
+
+We support Windows and Linux. The toolbox may be used as a **Matlab stand-alone library** or **stand-alone application** (**No Matlab is required**).
 
-# Install
+For a **stand-alone library** usage, **Matlab R2018b** and the Matlab Component Runtime (v95) or newer is required ( since version 2.6.1).
```
Comment (ggalibert) [inline README.md]: You don't need the MCR for use with Matlab.

Comment (ocehugo) [reply README.md]: oops - done

**Comment by ocehugo** [comment]
@ggalibert done, you can merge

**Pull request merged**